bin/ns-bundle

```diff
@@ -225,7 +225,7 @@ function spawnChildProcess(command, ...args) {
 
         const childProcess = spawn(command, escapedArgs, {
```

Comment:
Should spawn `tns run ios|android --bunlde ...` and obsolete the implementation.